_studio/mfx_lib/shared/include/mfx_h264_encode_vaapi.h

```diff
@@ -238,6 +243,7 @@ namespace MfxHwH264Encode
         VABufferID m_rirId;                     // VAEncMiscParameterRIR
         VABufferID m_qualityParamsId;           // VAEncMiscParameterEncQuality
         VABufferID m_miscParameterSkipBufferId; // VAEncMiscParameterSkipFrame
+        VABufferID m_roundingOffsetId;          // VAEncMiscParameterCustomRoundingControl
```

Comment:
Please, add initialization of this field to constructor

Reply:
OK, will add it.